### PR TITLE
Project: Restaurant Page

### PR DESCRIPTION
**Pull request by cvilla714**
In this project, I was able to implement  ES6 class keyword, ES6 modules, and webpack. 
How to use webpack to bundle all of my scss and js files to bundle them and convert them into one single file for managing all of my styles and also one single js file as well.

The main goal of this project was for me to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.

### CODE REVIEW

File: src/about.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <i class="fas fa-star-of-life star"></i>
+      <i class="fas fa-star-of-life star"></i>
+    </div>
+    <button type="button" class="main-btn">Read More</button>
+  </div>
+</section>
+  
+        `;
+
+  contentabout.appendChild(about);
```
Comment (peterrobert) [inline src/about.js]: You should use javascript only to create your dom. Use `document.createElement` and `
document.appendChild` to create your dom elements and append them.   You can learn more about this [here](https://www.w3schools.com/jsref/met_document_createelement.asp).

**NB:**  Note that there should be no HTML everything should be built in javascript. This includes all other pages in your application.

append_requirement

File: src/page_load.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+// import _ from 'lodash';
+// import "../css/style.css";
+// import Icon from "../src/images/nav-img-1.jpeg";
+
```
Comment (peterrobert) [inline src/page_load.js]: Remove commented code. This will keep your code clean

File: package.json [hunk 3]
```diff
@@ -1,16 +1,33 @@
...
   "version": "1.0.0",
+  "private": true,
   "description": "Restaurant Page",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "compile:sass": "node-sass sass/main.scss css/style.css -w",
+    "build": "webpack",
+    "start": "webpack serve --open"
   },
```
Comment (peterrobert) [inline package.json]: Great work adding this dependency.  However, you should also add a `dev` dependency:

example: ``"dev": "webpack --mode development",``

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
Hello, @cvilla714 very nice work. Good work modularising your code 👍🏿. However, there are a few changes to be made, kindly check below ⬇️ 

- [x] Your readme file should be descriptive. You have added a readme file but you didn't tell anything about the project. Remember this is a general requirement in all the projects you do. ⬇️ 
descriptive_readme

- [x] The images in your application are broken. The site itself appears broken. If I check chrome developers tools, your paths appear broken kindly fix this issue. ⬇️ 
Screenshot from 2021-01-29 08-26-30

- [x] Remember you need to create a tab menu to navigate through your pages. Kindly do so.

### if you have any question, kindly reach me via slack @peterrobert

### HAPPY CODING 💻

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Change Required❌.

## Hi @cvilla714 

#### Good job on removing commented code,addressing broken links in readme, appending all DOM element to `<div id="content"></div>` however, there are a few issues we have to address to get the project approved🤗.

* [x] Adding commands to script of `package.json` enables you run command `npm + "command in script"` these command enable you launch your easily etc... and ease development. In the previous review you asked to add `dev` command to script. Let us address this issue.

* [x] You are required not to use HTML syntax at all in this project, because, the lesson seeks to entrench your proficiency in javaScript DOM manipulation. As mentioned in the previous review, let us make sure all elements are created with JS. Use JS syntax only.

> you page is very beautiful by the way.

#### That is it kindly make the change and submit a re-review🤗.

#### Happy coding
Addo Forison

**Comment by cvilla714** [comment]
> # Status::Change Required❌.
> ## Hi @cvilla714
> #### Good job on removing commented code,addressing broken links in readme, appending all DOM element to `<div id="content"></div>` however, there are a few issues we have to address to get the project approved🤗.
> * [ ]  Adding commands to script of `package.json` enables you run command `npm + "command in script"` these command enable you launch your easily etc... and ease development. In the previous review you asked to add `dev` command to script. Let us address this issue.
> * [ ]  You are required not to use HTML syntax at all in this project, because, the lesson seeks to entrench your proficiency in javaScript DOM manipulation. As mentioned in the previous review, let us make sure all elements are created with JS. Use JS syntax only.
> 
> > you page is very beautiful by the way.
> 
> #### That is it kindly make the change and submit a re-review🤗.
> #### Happy coding
> Addo Forison

**Comment by cvilla714** [comment]
I need to make a few comments the project was done using pure javascript, I implemented Object Literal Syntax Extensions in ES6 to create the project, please take a look at the files individually and you can see that I am creating elements using javascript and then using object literals and appending the elements to the div content.

Here is what I mean 
```
const aboutLoad = (() => {
  // container
  const contentabout = document.querySelector("#content");
  const about = document.createElement("tabs");
  about.innerHTML = `
  <section class="about-us">
  <div class="about-us-left">
    <img src="../src/images/about-us-img.png" />
  </div>

  <div class="about-us-right">
    <h1 class="main-heading">About Us</h1>
    <h3 class="sub-heading">Introduce to Georgian Dishes</h3>
    <div class="stars">
      <i class="fas fa-star-of-life star"></i>
      <i class="fas fa-star-of-life star"></i>
      <i class="fas fa-star-of-life star"></i>
    </div>
    <p class="description">
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aliquam voluptatum, reiciendis repellat sapiente quisquam, voluptates expedita hic aperiam ipsam tempora ex temporibus corrupti assumenda laboriosam libero. Perspiciatis iste harum unde voluptatem fugiat nobis suscipit corporis nemo
      cum sapiente rem, praesentium mollitia, reiciendis temporibus quas, optio et id provident non cupiditate.
    </p>
    <div class="stars">
      <i class="fas fa-star-of-life star"></i>
      <i class="fas fa-star-of-life star"></i>
      <i class="fas fa-star-of-life star"></i>
    </div>
    <button type="button" class="main-btn">Read More</button>
  </div>
</section>
        `;

  contentabout.appendChild(about);
})();
export default aboutLoad;
```
as you can see I am creating the elements then adding object literals syntax from ES6 to add the content I need and finally appending it to the content.

If you check the repo it says 100% javascript it doesn't have any HTML .
2021-01-29_13h26_55

2021-01-29_13h16_27

2021-01-29_13h17_37

also inside my webpack.config.js I have the developer mode included 
2021-01-29_13h23_45

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

Well done @cvilla714, I loved your design.

now you can merge your project is approved :checkered_flag: 

## Happy Coding :tophat:

**Pull request merged**